extensions/wikia/AdEngine/js/video/googleIma.js

```diff
@@ -30,9 +33,26 @@ define('ext.wikia.adEngine.video.googleIma', [
 		return adsRequest;
 	}
 
+	function createProgressBar() {
+		var progressBar = doc.createElement('div'),
+			currentTime = doc.createElement('div');
+
+		progressBar.classList.add('progress-bar');
+		currentTime.classList.add('current-time');
+
+		progressBar.appendChild(currentTime);
+
+		return progressBar;
+	}
+
 	function prepareVideoAdContainer(videoAdContainer) {
+		var progressBar = createProgressBar();
+
 		videoAdContainer.style.position = 'relative';
```

Comment:
Oh, and it sets it inline, so that's why we can't overwrite it with a CSS class. Thanks!